clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/description/UpsertGoogleLoadBalancerDescription.groovy

```diff
@@ -26,6 +29,8 @@ class UpsertGoogleLoadBalancerDescription extends AbstractGoogleCredentialsDescr
   String portRange
   String region
   String accountName
+  GoogleHttpLoadBalancer googleHttpLoadBalancer
```

Comment:
The L7 description needs the layout of the backend services in the URL map, which is encoded in the `GoogleHttpLoadBalancer`. We could unpack that into its constituent `healthChecks`, `backendServices` and `hostRules` etc, but we'd be duplicating the structure of the `GoogleHttpLoadBalancer` object.